### PR TITLE
Bootstrap project

### PR DESCRIPTION
**Pull request by ooctavian**

### CODE REVIEW

File: domain/profile/query/querier.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+UPDATE profile
+SET alias = pggen.arg('alias'),
+    updated_at = NOW()
+WHERE account_id = pggen.arg('account_id');
+
+-- name: UpdateName :exec
+UPDATE profile
+SET name = pggen.arg('name'),
+    updated_at = NOW()
+WHERE account_id = pggen.arg('account_id');
```
Comment (louleeah) [inline domain/profile/query/querier.sql:23]: You don't need 2 different functions here.

File: domain/profile/query/querier.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
+-- name: GetProfile :one
+SELECT *
+FROM profile
+WHERE account_id = pggen.arg('account_id');
+
+-- name: CreateProfile :exec
+INSERT INTO profile(account_id, name)
```
Comment (louleeah) [inline domain/profile/query/querier.sql:7]: Please add "alias" as well.

File: domain/resource/service/create.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+		logrus.WithContext(ctx).
+			Errorf("Failed inserting in database: %v", err)
+	}
+
+	return &model.ResourceResponse{
+		ResourceId: res,
+	}, err
+}
+
+func (s *Resource) validateCategoryOwner(ctx context.Context, categoryId int, accountId int) error {
```
Comment (louleeah) [inline domain/resource/service/create.go:43]: Why are categories owned by specific accounts?

**Review by louleeah — CHANGES_REQUESTED** [review]

File: domain/resource/service/create.go (hunk 3 above)
Comment (ooctavian) [reply domain/resource/service/create.go:43]: Well, if we don't, then any account can modify/delete categories or they shouldn't be modifiable/deletable at all? If we let anyone use the category but not to edit/remove then what happens when the owner does it ?

Comment (ooctavian) [reply domain/resource/service/create.go]: Deleted category validation in new update

**Pull request merged**